Makefile

```diff
@@ -36,9 +36,9 @@ deploy:
 ifeq ($(POLICY_ENGINE), rego)
 	helm install -n gatekeeper-system gatekeeper gatekeeper/gatekeeper --create-namespace --version $(GATEKEEPER_VERSION) --set enableK8sNativeValidation=false
 else ifeq ($(POLICY_ENGINE), cel)
-ifneq ($(GATEKEEPER_VERSION), 3.15.1)
 	helm install -n gatekeeper-system gatekeeper gatekeeper/gatekeeper --create-namespace --version $(GATEKEEPER_VERSION) --set enableK8sNativeValidation=true
-endif
+else ifeq ($(POLICY_ENGINE), vap)
```

Comment:
For these, since Gatekeeper webhook is a fallback for VAP, how do we ensure the failure resulted from VAP instead of the GK webhook?

Reply:
We check the violation string to make sure it includes `ValidatingAdmissionPolicy` - https://github.com/open-policy-agent/gatekeeper-library/pull/618/files#diff-3b0266d06db240d302bf9febd17c09cf104dccce0bb2ee8e799e49c85f7082ffR96.